src/cai_lab_to_nwb/zaki_2024/zaki_2024_convert_week_session.py

```diff
@@ -49,6 +47,19 @@ def session_to_nwb(
     )
     conversion_options.update(dict(MultiEDFSignals=dict(stub_test=stub_test)))
 
+    # Add Cross session cell registration
+    main_folder = data_dir_path / f"/Ca_EEG_Calcium/{subject_id}/SpatialFootprints"
+    file_paths = []
+    for folder in os.listdir(main_folder):
+        folder_path = os.path.join(main_folder, folder)
+        if os.path.isdir(folder_path):  # Ensure it's a directory
```

Comment:
Just for your own info, you can also use pathlib.iterdir for these cases.

Reply:
right, thanks